site/content/docs/5.3/utilities/colors.md

```diff
@@ -46,16 +53,83 @@ Color utilities like `.text-*` that generated from our original `$theme-colors`
 **Deprecation:** With the addition of the expanded theme colors and variables, the `.text-muted` utility has been deprecated as of v5.3.0. Its default value has also been reassigned to the new `--bs-secondary-color` CSS variable to better support color modes. It will be removed in v6.0.0.
 {{< /callout >}}
 
-{{< callout info >}}
-For each `.text-*` there is a matching `.text-*-emphasis` utility. In Boosted, they have exactly the same value so we decided not to display them in the example above so that you don't hesitate on which class to use.
+<!-- Boosted mod -->
+## Orange's colors
 
-Here is a list of these extra classes:
-{{< colors.inline >}}
-{{- range (index $.Site.Data "theme-colors") }}
-- `.text-{{ .name }}-emphasis`
-{{- end -}}
-{{< /colors.inline >}}
-{{< /callout >}}
+Bootstrap provides many `.text-*` and `.bg-*` utilities, but they should be used with care to meet our design specifications and [WCAG 2.0 accessibility standards for color contrast](https://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-contrast.html).
+To be sure to respect the specifications, it is necessary to define `color`, `background-color` and `font-size` altogether.
+
+Thus, the `.text-primary` color on white background (`#f16e00`) can only be used in a font size greater than 24px (using for example `.fs-3` utility), or 19px bold (using for example `.fs-4` and `.fw-bold` utilities).
+The `.text-primary` color on dark background (`#ff7900`) can be used in any size, and it shouldn't be used on light grey backgrounds at all.
+
+Here are some compliant combinations examples for texts:
+
+{{< example >}}
+
+<p class="text-black">regular black text</p>
+<p class="text-primary fs-3">regular primary text with minimum font-size for contrast with .fs-3</p>
+<p class="text-body-secondary">regular secondary text</p>
+<p class="text-light">regular disabled text</p>
```

Comment:
This one bothers me a little. OK `.text-light` color has the same value as the disabled color, but `.text-light` is not really the disabled color. In dark mode, `.text-light` will keep the same color while the disabled color might change value. So this text won't be valid anymore and maybe projects would have to use `.text-light` everywhere for disabled stuff and would need to revert it.

We can see that `.text-body-secondary` is used in the following block to display "regular disabled text on dark". Folks won't switch classes depending on light/dark mode to display disabled content. They would use something like `.text-disabled` (which doesn't exist) or set the color in CSS.

I'm not 100% sure yet but I would get rid of `.text-light` here, `.text-body-secondary` in the `.bg-dark` block, and the disabled icon (+ note) which is the same case.
Otherwise, keep them but do not explain the disabled rendering 🤷 

Thoughts?

Reply:
Yes maybe its more simple this way for users, if we don't talk about disabled color... And they still have the classes `.text-light` and `.text-body-secondary` above, in the section "Colors", so I think we can remove them totally from "Orange's colors"